UserManagement/accounts/tests/test_domain_models.py

```diff
@@ -0,0 +1,70 @@
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+from django.db.utils import IntegrityError
+from ..domain.models import UserProfile, InstructorRate
+from datetime import date, datetime
+from decimal import Decimal
+
+
+class UserProfileTestCase(TestCase):
+    def setUp(self):
+        UserProfile.objects.create(identity_id=123, full_name="John Doe", date_of_birth="1990-01-01")
+
+    def test_user_creation(self):
```

Comment:
name conventions - [MethodUnderTest]_[TestedState]_[ExpectedBehavior] @kossvat

Reply:
for all test

Reply:
or like this 

example - def test_extract_name_when_present(...):
    ...

def test_extract_name_fails_when_missing(...):
    ...